optfiles/linux_amd64_ifort+mpi_stampede2_skx

```diff
@@ -21,6 +21,8 @@ NOOPTFLAGS='-O1 -fp-model precise'
 
 FFLAGS="$FFLAGS -W0 -WB -convert big_endian -assume byterecl"
 FFLAGS="$FFLAGS -fPIC"
+FFLAGS="$FFLAGS -mcmodel=large -shared-intel"
```

Comment:
I don't have `-shared-intel`

Reply:
I had to remind myself here ... this is just to be explicit. According to [the docs](https://software.intel.com/content/www/us/en/develop/documentation/fortran-compiler-oneapi-dev-guide-and-reference/top/compiler-reference/compiler-options/compiler-option-details/linking-or-linker-options/shared-intel.html)

> If you specify option -mcmodel=medium or -mcmodel=large, it sets option -shared-intel.

so it doesn't really matter, but I'll just keep it in